microshift_install/microshift-install-rpm.adoc

```diff
@@ -6,3 +6,31 @@ include::_attributes/attributes-microshift.adoc[]
 toc::[]
 
 // The content of this file should be based on https://github.com/openshift/microshift/blob/main/docs/devenv_rhel8.md but without the steps for building from source.
+
+You can install MicroShift from an RPM on a virtual machine with {op-system-first} 8, or on a {op-system} production-similar environment. 
+
+== Installing MicroShift from RPM
+
+Use the following procedures to create a MicroShift development environment on {op-system-first} 8.7 or greater by following the content in this section. 
```

Comment:
We have to be careful about version numbers right now. It's only supported on 8.7, not newer versions of 8. We may move to 9 in the next release and drop support for 8.